### PR TITLE
Address nits from PR #3423

### PR DESCRIPTION
**Pull request by saad-ali**
Address nits from PR #3423

### CODE REVIEW

**Comment by erictune** [comment]
LGTM

File: pkg/kubelet/config/config.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func (c *PodConfig) Channel(source string) chan<- interface{} {
 	return c.mux.Channel(source)
 }
 
-func (c *PodConfig) SourceSeen(source string) bool {
+func (c *PodConfig) IsSourceSeen(source string) bool {
```
Comment (lavalamp) [inline pkg/kubelet/config/config.go]: If we're fixing nits... please add a godoc comment, since this is a public function :)

Comment (saad-ali) [inline pkg/kubelet/config/config.go]: Done.

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**